### ISSUE TITLE
Extractor condition "Field contains string" uses wrong tester API

### ISSUE THREAD

**Issue by dbinoj** [Graylog2/graylog2-server#3134]
The "Try" button of "Field contains string" field of an extractor when "Only attempt extraction if field contains string" condition is selected expects special chars to be escaped. But the actual extractor does not need these chars to be escaped.

## Expected Behavior
If I want to run an extractor only on messages containing a `{`, I would set "Field contains string" to `{` and click "Try" button. I expect `Matches! Extractor would run against this example.` to be displayed below the field.

## Current Behavior
When I click "Try" with `{`, I get:
```
Could not try regular expression. Make sure that it is valid.
Details: Error: cannot POST https://graylog/api/tools/regex_tester (400)
```

So, I tried to escape the `{` with `\{`. Now, the "Try" button works. *But*, the extractor itself fails to run on messages containing `{`. I have to set "Field contains string" to `{` and ignore the error when I "Try" for the extractor to work.

## Steps to Reproduce (for bugs)
1. Send a log message which contains `{`
2. Create an extractor and try to "Only attempt extraction if field contains string `{` and click the "Try" button.
3. Create another extractor and change the "Field contains string" field to `\{` and click the "Try" button.
5. Try to parse messages containing `{` with both extractors.

## Context
Have a filed which sends `null` or a JSON payload. I wanted the extractor not to run when the field content was `null`.

## Environment

* Graylog Version: 2.1.2+50e449a
* Operating System: CentOS / Oracle Corporation 1.8.0_73 on Linux 3.10.0-327.36.2.el7.x86_64
* Browser version: Chrome 54.0.2840.99 m (64-bit)

**Comment by kroepke** [comment]
@dbinoj Could you please post your exact extractor configuration so we can debug this, please?

**Comment by dbinoj** [comment]
I'm attaching screenshots.
Try fails, but extractor works:
![try fail](https://cloud.githubusercontent.com/assets/831913/20883213/f17a39fc-bb0a-11e6-8db6-e85f783f7084.PNG)
Try passes, but extractor does not work:
![try pass](https://cloud.githubusercontent.com/assets/831913/20883212/f1780056-bb0a-11e6-9b1f-7d68d4219e01.PNG)

**Comment by kroepke** [comment]
@dbinoj Cool thanks.
Looks like the frontend uses the wrong tester URL to validate the condition value (instead of using the string contains one it uses the regex tester, which of course fails).

So you want `{` which makes the tester fail right now, but at runtime it should perform the action you want.
We'll fix the tester for 2.2.

**Comment by dbinoj** [comment]
Thanks!
